src/to_wkt/mod.rs

```diff
@@ -1,5 +1,7 @@
 use crate::{Wkt, WktNum};
 
+pub mod geo_trait_impl;
```

Comment:
I don't love the name of this module. Especially if it's becoming the de-facto writer.

Reply:
In case you're copying this naming convention from the `geojson` crate pull request, I went with this name knowing that it wouldn't be exposed publicly. I didn't put a lot of thought into it so feel free to stray from it

Reply:
I think this really depends on https://github.com/georust/geo/pull/1241 and/or requiring  a `geo-types` dependency here. With this PR _all_ writing goes through the trait impl, but the traits currently depend on `geo-types`.

In that case, we could make `geo_trait_impl` private and re-export the functions through `wkt::to_wkt::*`

Reply:
ref https://github.com/georust/wkt/pull/124/commits/f3434f5283d070f1accd64cd1bd09303f0f0547c